### ISSUE TITLE
Can't install submodules

### ISSUE THREAD

**Issue by fletcherist** [nperraud/gan_audio_inpainting#1]
Heya! Thanks for the great article and repo. I'm trying to learn how it all works. I have tried to setup the repo, but something went wrong

in `README.md` it's said to install submodules by using
```bash
git submodule update --init --recursive
```
however when I try that, I get the following error:
```
Submodule 'code/gantools' (git@github.com:nperraud/gantools.git) registered for path 'gantools'
Cloning into '/root/gan_audio_inpainting/code/gantools'...
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:nperraud/gantools.git' into submodule path '/root/gan_audio_inpainting/code/gantools' failed
Failed to clone 'code/gantools'. Retry scheduled
Cloning into '/root/gan_audio_inpainting/code/gantools'...
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:nperraud/gantools.git' into submodule path '/root/gan_audio_inpainting/code/gantools' failed
Failed to clone 'code/gantools' a second time, aborting
```

**Comment by fletcherist** [comment]
I changed `git@github.com:nperraud/gantools.git` to `https://github.com/nperraud/gantools.git` in `.gitmodule` and now i have this:

```Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:nperraud/tfnntools.git' into submodule path '/root/gan_audio_inpainting/code/gantools/tfnntools' failed
Failed to clone 'tfnntools'. Retry scheduled
Cloning into '/root/gan_audio_inpainting/code/gantools/tfnntools'...
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights```

the same thing but with your another library

**Comment by nperraud** [comment]
There is a submodule inside the submodule. You can do the same thing there... We should probably change everything to https...

**Comment by nperraud** [comment]
This has been fixed in the commit 6113d12
Thanks for reporting

**Comment by fletcherist** [comment]
thanks. it works now!
